Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM python:2.7.16-slim-stretch
+
+ENV iraf /iraf/iraf/
+ENV IRAFARCH linux64
+ENV LCOSNDIR /supernova
+ENV LCOSNPIPE $LCOSNDIR/github/lcogtsnpipe
+ENV DISPLAY host.docker.internal:0
+
+RUN apt-get update \
+        && apt -y install gcc make flex git \
+        && apt -y install libcurl4-openssl-dev libexpat-dev libreadline-dev \
+        && apt-get autoclean \
+        && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p $iraf \
+        && cd /iraf \
+        && git clone https://github.com/iraf-community/iraf.git \
+        && cd $iraf \
+        && git checkout 567961f \
+        && ./install < /dev/null \
+        && make linux64 \
+        && make sysgen
+
+RUN apt-get update \
+        && apt-get -y install libx11-dev libcfitsio-bin wget x11-apps libtk8.6 sextractor \
+        mysql-client libmariadbclient-dev openssh-client wcstools libxml2 vim libssl1.0.2 zip \
+        && apt-get autoclean \
+        && rm -rf /var/lib/apt/lists/*
+
+RUN ln -s /usr/bin/sextractor /usr/bin/sex
+
+RUN pip install numpy>=1.12 astropy matplotlib pyraf mysql-python scipy astroquery statsmodels==0.10 cython
+
+RUN pip install git+git://github.com/kbarbary/sep.git@master git+git://github.com/dguevel/PyZOGY.git && rm -rf ~/.cache/pip
+
+RUN wget http://ds9.si.edu/download/debian9/ds9.debian9.8.0.1.tar.gz \
+        && tar -xzvf ds9.debian9.8.0.1.tar.gz -C /usr/local/bin \
+        && rm -rf ds9.debian9.8.0.1.tar.gz
+
+RUN mkdir -p /home/supernova/iraf && /usr/sbin/groupadd -g 20000 "domainusers" \
+        && /usr/sbin/useradd -g 20000 -d /home/supernova -M -N -u 10197 supernova \
+        && chown -R supernova:domainusers /home/supernova \
+        && mkdir -p $LCOSNDIR
+
+RUN wget http://cdsarc.u-strasbg.fr/ftp/pub/sw/cdsclient.tar.gz \
+        && tar -xzvf cdsclient.tar.gz -C /usr/src && rm cdsclient.tar.gz \
+        && cd /usr/src/cdsclient-* && ./configure && make && make install
+
+COPY ./configure $LCOSNDIR
+
+COPY . $LCOSNPIPE
+
+RUN chown -R supernova:domainusers $LCOSNDIR /usr/local
+
+USER supernova
+
+WORKDIR $LCOSNPIPE/trunk
+
+RUN python setup.py install
+
+#RUN ln -s $LCOSNDIR/data/apass/ $LCOSNDIR/data/sloan/ /usr/local/lib/python2.7/site-packages/lsc/standard/cat/
```

Comment:
can someone explain why this is commented out?

Reply:
looks like it was from @abostroem's commit

Reply:
When I built the docker image this I got this error message:
```
Removing intermediate container a752516a6ec8
 ---> 8893da0b3c97
Step 21/25 : RUN ln -s $LCOSNDIR/data/apass/ $LCOSNDIR/data/sloan/ /usr/local/lib/python2.7/site-packages/lsc/standard/cat/
 ---> Running in 941c3e3ae4db
ln: failed to create symbolic link '/usr/local/lib/python2.7/site-packages/lsc/standard/cat/apass': File exists
ln: failed to create symbolic link '/usr/local/lib/python2.7/site-packages/lsc/standard/cat/sloan': File exists
The command '/bin/sh -c ln -s $LCOSNDIR/data/apass/ $LCOSNDIR/data/sloan/ /usr/local/lib/python2.7/site-packages/lsc/standard/cat/' returned a non-zero code: 1
```

Reply:
I think this happens when you have used the git repo previously and there are existing catalogs. My commit below moves these out of the way first and then does the linking.